app/models/intake.rb

```diff
@@ -276,6 +280,10 @@ def self.searchable_fields
 
   before_save do
     self.needs_to_flush_searchable_data_set_at = Time.current
+    if email_address.present?
```

Comment:
Will this cause unnecessary writes? i.e., when saving irrelevant columns, will Postgres try to update the `self.email_domain` column?